src/actors/upgrade/vcs_repos/vcs_detection/check

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+
+import sys
+import json
+
+
+def load_inputs():
+    "Load all inputs and return dictionary with all input channels."
+    inputs = {}
+    if not sys.stdin.isatty():
+        input_data = sys.stdin.read()
+        if input_data:
+            inputs = json.loads(input_data)
+    return inputs
+
+
+def store_output(channel, data):
+    "Store the data into the expected channel (inside outputs object)."
+    # TODO: create class for input/output data? of course, this particular
+    # function works just for StringList now
+    outputs[channel] = [{"value": data}]
+    return
```

Comment:
if you want to be really explicit, you should ```return None```, although I would remove this return. 

Anyway what is the reason behind to use the function to update dict?
I know you can modify mutable objects inside functions without declaring globals, but i think this is bad idea.
(you're updating ```outputs``` which is defined somwhere else)

If you want to build output dict by using function, I would do this in different way, like:

```
def build_output(vcs):
    try:
        return {"vcs_repo_%s" % repo: data for repo, data in vcs.items()}
    except TypeError as exc:
         log_error(exc.message)  # you could also add here your message 
         return {}
```
then I would do
 ```
output = build_output(vcs)
print(json.dumps(output))
```